packages/core/src/locales/ar.ts

```diff
@@ -53,66 +53,65 @@ const Nouns: {
   ulid: "ULID",
   xid: "XID",
   ksuid: "KSUID",
-  datetime: "ISO datetime",
-  date: "ISO date",
-  time: "ISO time",
-  duration: "ISO duration",
-  ipv4: "IPv4 address",
-  ipv6: "IPv6 address",
-  cidrv4: "IPv4 range",
-  cidrv6: "IPv6 range",
-  base64: "base64-encoded string",
-  base64url: "base64url-encoded string",
-  json_string: "JSON string",
-  e164: "E.164 number",
+  datetime: "تاريخ ووقت بمعيار ISO",
+  date: "تاريخ بمعيار ISO",
+  time: "وقت بمعيار ISO",
+  duration: "مدة بمعيار ISO",
+  ipv4: "عنوان IPv4",
+  ipv6: "عنوان IPv6",
+  cidrv4: "مدى عناوين بصيغة IPv4",
+  cidrv6: "مدى عناوين بصيغة IPv6",
+  base64: "نَص بترميز base64-encoded",
+  base64url: "نَص بترميز base64url-encoded",
+  json_string: "نَص على هيئة JSON",
+  e164: "رقم هاتف بمعيار E.164",
   jwt: "JWT",
-  template_literal: "input",
+  template_literal: "نص مقولب",
```

Comment:
Same here, "مدخل" would be better (or any other translation for "input").
```suggestion
  template_literal: "مدخل",
```